### PR TITLE
bug: update _episodes/04-metadata.md

### LINKED ISSUE #16: Update _episodes/04-metadata.md
- [x] This should probably come before "other files" if we're using the "specific file, specific file, everything else" pattern. Decide how to handle that. 
- [x] Make sure the commands to do the packaging are included in the overview: "these are all the files you need, we'll look at what needs to go in each, these are the commands you'll need for testing, packaging, publishing, etc."

### PR DESCRIPTION
**Pull request by hollandjg**
- Put the top overview into bullet-point form

Resolutions
- resolves #16

### CODE REVIEW

File: _episodes/04-metadata.md [hunk 1]
```diff
@@ -116,9 +126,17 @@ classifiers = [
     "Topic :: Scientific/Engineering :: Mathematics",
     "Topic :: Scientific/Engineering :: Physics",
     "Typing :: Typed",
+    "Private :: Do Not Upload",
 ]
 ```
 
+> ## Prevent Inadvertent Publishing
+> By adding the "Private :: Do Not Upload" trove classifier here, we ensure that
```
Comment (gtdang) [inline _episodes/04-metadata.md:134]: What is a "trove" classifier? Perhaps the previous section should be renamed to be consistent with how classifiers are referred to in the rest of the document.  Alternatively it should be defined on explained why they are referred to as "trove classifiers".

File: _episodes/04-metadata.md [hunk 2]
```diff
@@ -116,9 +126,17 @@ classifiers = [
...
+> the package will be
+> [rejected when we try to upload it to PyPI](https://pypi.org/classifiers/#:~:text=To%20prevent%20a%20package%20from,beginning%20with%20%22Private%20%3A%3A%22.).
+> If you want to upload to PyPI, you will need to remove that classifier.
+{:.callout}
+
 ### License (special mention)
 
 There also is a license field, but that was rather inadequate; it didn't support
```
Comment (gtdang) [inline _episodes/04-metadata.md:142]: The text in the license section is a bit confusing. The first sentence mixes up tenses. Is it still inadequate? 

The sentence "There's work on standardizing an update to the format in the future." is also confusing. Does it mean there's "There is current work to standardize the format."?  We should also check on if this statement is still true. Curious what the progress is and what they plan to standardize (ways to specify multiple licenses?).

File: _episodes/04-metadata.md (hunk 1 above)
Comment (hollandjg) [reply _episodes/04-metadata.md]: I've updated this section to include an etymology for the "Trove" part, and made everywhere else use the full proper noun "Trove Classifier". Does that make it clearer?

Comment (hollandjg) [reply _episodes/04-metadata.md]: I just looked here, and python.org doesn't say "Trove Classifiers" any more – they just say PyPI classifiers
https://packaging.python.org/en/latest/guides/writing-pyproject-toml/. Shall we drop the "Trove" bit?

File: _episodes/04-metadata.md (hunk 2 above)
Comment (hollandjg) [reply _episodes/04-metadata.md]: I'm not sure – I've included a new version of this with a simpler set of recommendations.

File: _episodes/04-metadata.md (hunk 1 above)
Comment (gtdang) [reply _episodes/04-metadata.md]: Yeah I support simplifying the language to just Classifiers.

File: _episodes/04-metadata.md (hunk 2 above)
Comment (gtdang) [reply _episodes/04-metadata.md]: Great! I like your changes.

File: _episodes/04-metadata.md (hunk 1 above)
Comment (hollandjg) [reply _episodes/04-metadata.md]: Done

**Pull request merged**